src/components/ComputeModal.js

```diff
@@ -783,6 +783,9 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
       ],
       () => ({ disabled: !hasChanges() || !!errors, tooltip: Utils.summarizeErrors(errors) })
     )
+
+    const isUpdateDisabled = getIsRuntimeBusy(currentRuntimeDetails)
```

Comment:
this is a moderate paradigm-shift that's not captured in bullet points at the top. Essentially, we want to start shifting responsibility on the compute modal to disable updating/deleting if the runtime is in a 'busy' state, and allow the modal to open in a read-only state still. Today, we disable viewing the compute configuration entirely and don't offer this read-only view.